spec/unit/base/RequestClient.spec.js

```diff
@@ -46,6 +46,11 @@ describe('lastResponse and lastRequest defined', function() {
     expect(client.lastResponse.body).toEqual('voltron');
   });
 
+  it('should not include request authorization header in filtered headers', function () {
+    const filteredHeaderKeys = client.filterLoggingHeaders(client.lastRequest.headers);
+    expect(filteredHeaderKeys).toEqual(['test-header-key', 'Connection'])
```

Comment:
There's no authorization header to begin with. Should add this to the test data.

Reply:
`client.lastRequest.headers` includes the test-header-key, connection, and authorization header when it mocks the request. Is there a additional authorization header that needs to be tested that does not take the format of `Authorization: 'Basic abcd'
` that I need to include in the test data?

Reply:
Ah, never mind then.